_posts/2020-03-31-April-open-source-report.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: April-open-source-report
+date: 2020-03-31 20:12:12
+categories:
+author: bronzdoc
+
+---
+
+# Ombulabs Open Source report
+
+![img](https://external-preview.redd.it/zbCp6e3VikD9vsKOtTGVKI90UnWIOZN5NMRNSvgvXxE.jpg?width=960&crop=smart&auto=webp&s=9318c01ae704565ad90f4736aec5af6b671d98fa)
+
+# Relevant News
+
+## Skunk
+
+[Skunk](https://github.com/fastruby/skunk) Is a RubyCritic extension to calculate StinkScore for a file or project.
+
+From Jan 01, 2020 through Apr 05, 2020, fastruby/skunk gained 58 new commits, contributed by 5 authors. There were 506 additions and 269 deletions across 26 files.
+
+New Contributors
+There were 2 new contributors during this report period.
+
+Grzegorz Jakubiak
+Sebastián Palma
+Pull Requests
+A total of 11 pull requests were opened during this period.
+
+Average age: approximately 1 month and 3 weeks.
+Oldest pull request was opened on Jan 30, 2020.
+Newest pull request was opened on Feb 29, 2020.
+Issues
+A total of 3 issues were opened during this period.
+
+Average age: approximately 1 month and 3 weeks.
+Oldest issue was opened on Jan 14, 2020.
+Newest issue was opened on Mar 10, 2020.
+
+### New Logo:
+Skunk has a new logo and we also have stickers so make sure to ping [Ernesto](https://github.com/etagwerker) to give you some.
+
+![Imgur](https://i.imgur.com/LfeWWxK.png)
+
+
+## Bundler-leak
+
+[bundler-leak](https://github.com/rubymem/bundler-leak) Is a tool to find leaky gems in your dependencies.
+
+From Jan 01, 2020 through Apr 02, 2020, rubymem/bundler-leak gained 15 new commits, contributed by 4 authors. There were 148 additions and 76 deletions across 11 files.
+
+New Contributors
+There were 2 new contributors during this report period.
+
+Alexander Popov
+chubchenko
+Pull Requests
+A total of 5 pull requests were opened during this period.
+
+Average age: approximately 1 month and 3 weeks.
+Oldest pull request was opened on Jan 11, 2020.
+Newest pull request was opened on Feb 19, 2020.
+Issues
+A total of 0 issues were opened during this period.
+
+## Rubymem.com
+
+From Jan 01, 2020 through Apr 05, 2020, rubymem/rubymem.com gained 3 new commits, contributed by 2 authors. There were 3 additions and 3 deletions across 1 file.
+
+New Contributors
+There were no new contributors during this report period.
+
+Pull Requests
+A total of 2 pull requests were opened during this period.
+
+Average age: approximately 1 month and 6 days.
+Oldest pull request was opened on Feb 26, 2020.
+Newest pull request was opened on Mar 03, 2020.
+
+## Blogcop
```

Comment:
I don't remember if we wanted to display it like this:

```suggestion
## BlogCop
```

What do you think, @lubc?